.cico/cico_openshift_e2e.sh

```diff
@@ -9,41 +9,39 @@
 #
 set -ex
 
-# Setup to find necessary data from cluster setup
-export CI="openshift" # ENV used by PROW ci to get the oc client
+# ENV used by PROW ci
+export CI="openshift" 
 export ARTIFACTS_DIR="/tmp/artifacts"
-export CUSTOM_HOMEDIR=$ARTIFACTS_DIR
 
-# Overridable information
-export DEFAULT_INSTALLER_ASSETS_DIR=${DEFAULT_INSTALLER_ASSETS_DIR:-$(pwd)"/.e2e"}
-export KUBEADMIN_USER=${KUBEADMIN_USER:-"kubeadmin"}
-export KUBEADMIN_PASSWORD_FILE=${KUBEADMIN_PASSWORD_FILE:-"${DEFAULT_INSTALLER_ASSETS_DIR}/auth/kubeadmin-password"}
+# Pod created by openshift ci don't have user. Using this envs should avoid errors with git user.
+export GIT_COMMITTER_NAME="CI BOT"
+export GIT_COMMITTER_EMAIL="ci_bot@notused.com"
 
-# Exported to current env
-export KUBECONFIG=${KUBECONFIG:-"${DEFAULT_INSTALLER_ASSETS_DIR}/auth/kubeconfig"}
+# Check if operator-sdk is installed and if not install operator sdk in $GOPATH/bin dir
+if ! hash operator-sdk 2>/dev/null; then
+    mkdir -p $GOPATH/bin
+    export PATH="$PATH:$(pwd):$GOPATH/bin"
+    RELEASE_VERSION=v0.17.0
 
-# Check if file with kubeadmin password exist
-if [ ! -f $KUBEADMIN_PASSWORD_FILE ]; then
-    echo "Could not find kubeadmin password file"
-    exit 1
+    curl -LO https://github.com/operator-framework/operator-sdk/releases/download/${RELEASE_VERSION}/operator-sdk-${RELEASE_VERSION}-x86_64-linux-gnu
+
+    chmod +x operator-sdk-${RELEASE_VERSION}-x86_64-linux-gnu && \
+        cp operator-sdk-${RELEASE_VERSION}-x86_64-linux-gnu $GOPATH/bin/operator-sdk && \
+        rm operator-sdk-${RELEASE_VERSION}-x86_64-linux-gnu
 fi
 
-# Get kubeadmin password from file
-export KUBEADMIN_PASSWORD=$(cat $KUBEADMIN_PASSWORD_FILE)
+# Add kubernetes-api CRDS
+make update_devworkspace_crds
 
-set +x
-# Login as admin user
-oc login -u $KUBEADMIN_USER -p $KUBEADMIN_PASSWORD --insecure-skip-tls-verify
-set -x
+# Install go modules
+go mod tidy
+go mod vendor
```

Comment:
To make vendor folder used, you probably need to add `-mod=vendor` to go test command. But It's OK just download dependencies to be inlined in build in Dockerfile
```suggestion
go mod download
```